src/utils/GetFilteredPeople.ts

```diff
@@ -0,0 +1,64 @@
+import { Person } from '../types';
+
+interface FilterOptions {
+  query: string | null;
+  sex: string | null;
+  centuries: string[] | null;
+}
+
+interface SortOptions {
+  sort: string | null;
+  order: string | null;
+}
+
+function getSortedPeople(people: Person[], sortOptions: SortOptions) {
+  const { sort, order } = sortOptions;
+
+  const sorted = [...people].sort((p1, p2) => {
+    switch (sort) {
+      case 'name':
```

Comment:
The case for sorting by 'name' should match the `sortableColumns` definition, which uses 'Name'. Ensure consistency in naming conventions to avoid potential issues.